### DISCUSSION TITLE
Install Statamic standalone, or as a package in Laravel?

### DISCUSSION THREAD

**Discussion by calebdw** [statamic/cms#6887] [Q&A] [answered]
Hi there!

First time user of Statamic here, I'm looking to convert my church's WP site to Statamic (i.e., rebuild) and I had a question.

Is there any fundamental difference between 1) installing Statamic by itself through the CLI tool and 2) installing Laravel first, then Statamic as a package? While the website won't be too complicated I also don't want to necessarily pigeonhole myself if there's Pros/Cons to either approach.

Thanks!

**Comment by jackmcdade** [comment, marked as answer]
If you install it standalone you get a few things configured for you automatically (a pages collection, example homepage, etc). Installing to Laravel will also assume you’re using a database for your users, though you can easily switch that. On Oct 15, 2022, at 8:45 AM, CalebW ***@***.***> wrote:﻿
Hi there!
First time user of Statamic here, I'm looking to convert my church's WP site to Statamic (i.e., rebuild) and I had a question.
Is there any fundamental difference between 1) installing Statamic by itself through the CLI tool and 2) installing Laravel first, then Statamic as a package? While the website won't be too complicated I also don't want to necessarily pigeonhole myself if there's Pros/Cons to either approach.
Thanks!
